apps/api/src/app/events/e2e/map-trigger-recipients.e2e.ts

```diff
@@ -569,19 +582,21 @@ describe('MapTriggerRecipientsUseCase', () => {
       const command = buildCommand(session, transactionId, [
         secondTopicRecipient,
         firstRecipient,
-        firstSubscriber._id,
-        secondSubscriber._id,
+        firstSubscriber.subscriberId,
+        secondSubscriber.subscriberId,
         firstTopicRecipient,
         secondRecipient,
-        thirdSubscriber._id,
+        thirdSubscriber.subscriberId,
       ]);
       const result = await useCase.execute(command);
 
-      // We process first recipients that are not topics so they will take precedence when deduplicating
+      expect(result.length).to.equal(3);
```

Comment:
Looks like this test was not testing the reduplicating at all because the `to.include.deep.members` validated that the provided set exist in the expected object `result`, which was true but the result list was with other subscribers as well.
I am not sure if this was the expected behavior or not, but IMO the test should have tested the duplication `filtering` .